syntax/block_scanner.go

```diff
@@ -53,7 +53,7 @@ func (s *blockScanner) nextToken(outVal *tokenValue) Token {
 
 func (s *blockScanner) nextTokenInner() blockScannerToken {
 	if s.matchesNewBlock() {
-		return s.buildIndent()
+		return s.buildIndent(s.prevTokens[len(s.prevTokens)-1])
```

Comment:
we use `s.prevTokens[len(s.prevTokens)-1]` inside `buildIndent()` so i think we should keep `buildIndent()` argumentless.

Reply:
done